### PR TITLE
SALTO-1251 create workspace top-level cmd and move clean to it as sub…

### PR DESCRIPTION
**Pull request by emaohi**
Added new "workspace" command to the CLI to handle administrative commands. moved the "clean" command to be a sub-command of this new top-level command.

---
_Release Notes_: 
*CLI*
The `clean` admin command is now available as `workspace clean`

### CODE REVIEW

File: packages/cli/src/commands/workspace.ts [hunk 1]
```diff
@@ -121,7 +121,18 @@ const cleanDef = createWorkspaceCommand({
...
-  action,
+  action: cleanAction,
 })
 
-export default cleanDef
+// Group definition
+const wsGroupDef = createCommandGroupDef({
+  properties: {
+    name: 'workspace',
+    description: 'Workspace Admin commands',
```
Comment (netama) [inline packages/cli/src/commands/workspace.ts:131]: @tomermevorach @gilhoffer any preference for the description?

**Review by netama — APPROVED** [review]
LGTM 🎈 , let's wait for Gil or Tomer for the new command's description before merging

**Comment by netama** [comment]
nit on the release notes - this is user facing, maybe something like 
>The `clean` admin command is now available as `workspace clean`

(and please avoid abbreviations like `cmd` in these :) )

File: packages/cli/src/commands/workspace.ts (hunk 1 above)
Comment (tomermevorach) [reply packages/cli/src/commands/workspace.ts:131]: Nit: I'd rename it to 'Workspace administration commands'

**Pull request merged**